script/create-files

```diff
@@ -5,7 +5,7 @@
 # Source it up
 # shellcheck disable=SC1090
 source "$HOME/.trainingmanualrc"
-source "$(dirname "$0")/ask"
+source ask
```

Comment:
Should this be `./ask` not `ask`?

Reply:
> Unless you provide an exact path to filename then bash will look first via the PATH variable and then in the current directory (only if filename is not found in $PATH.) If any arguments are supplied, they become the positional parameters when filename is executed. Otherwise the positional parameters are unchanged. [[source](https://ss64.com/bash/source.html)]

While `ask` works, I do think `./ask` would be better in this case. Thanks, @beardofedu!